docs/authentication.md

```diff
@@ -36,6 +36,7 @@ token from your application's
 
 The following example creates an API client with a developer token:
 
+<!-- sample get_authorize -->
```

Comment:
I'm not sure this is really the right sample for that endpoint; this code doesn't make any API calls, but rather instantiates a client with a developer token

Reply:
It's the right sample for what I'd like to achieve: show how to init the SDK and get the auth URL.

This maps to the https://account.box.com/api/oauth2/authorize endpoint (`get_authorize`) which isn't really an endpoint but a redirect.

Reply:
That isn't what this piece of code does, though; it creates a client with a Developer Token and does not generate the authorize URL.  There's a fake redirect URL in the code sample, but it's not being used.  We don't have a sample in this case for what you're looking for.

Reply:
I'm happy to merge this for now. I am going through these endpoints soon in the next few weeks.